src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/toolbar_js.html.twig

```diff
@@ -14,6 +14,11 @@
             'sfwdt{{ token }}',
             '{{ path("_wdt", { "token": token }) }}',
             function(xhr, el) {
+
+                /* Evaluate embedded scripts inside the toolbar */
+                var rx=/<script>(.*?)<\/script>/g, s;
+                while (s = rx.exec(el.innerHTML)) {eval(s[1]);};
```

Comment:
This PR adds support to WebProfilerBundle in CSP context without `unsafe-inline`. `unsafe-eval` is not covered by this one.

I don't want to support the absence of `unsafe-eval` policy to the script-src directive in this PR. This is quite a big job:

We need some javascript to toggle nested levels in the DebugBundle toolbar block.
There are two way for doing this: `eval`'ing JS after XHR load, or register a callback in toolbar_js (without `eval`-.

The first is the way we're currently doing this, and probably the way any other implementors are doing at the moment.

The second way would need a deep change in the way the toolbar_js to allow Panels to register callbacks. This is a deep change because the event subscriber used to inject the toolbar does not have access to the TemplateManager at the moment.

I propose to work on this topic if some users report the issue.